vhdbuilder/packer/components.json

```diff
@@ -140,7 +140,8 @@
       "downloadURL": "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:*",
       "versions": [
         "ciprod03262021",
-        "ciprod04222021"
+        "ciprod04222021",
+        "ciprod05122021"
```

Comment:
Can you also help to add ciprod05122021 for Windows? Example: https://github.com/Azure/AgentBaker/pull/813/files

Reply:
We aren't publishing a windows agent with this release (no changes were made to it. We could republish the old windows image under a new tag, but we would prefer to keep using the old windows tag)

Reply:
Do you mean that ciprod04222021 is still used in AKS Windows nodes?

Reply:
Correct, ciprod04222021 is still the latest and greatest for Windows nodes.